app/services/solr_mapper.rb

```diff
@@ -69,7 +69,8 @@ def provider_identifier_field
   def doi_field
     return '' unless (identifier = metadata['identifiers'].find { |i| i['identifier_type'] == 'DOI' })
 
-    identifier['identifier']
+    # To standardize the DOI field string between sources, we want to remove any 'doi:' prefix
+    identifier['identifier'].delete_prefix('doi:')
```

Comment:
How about using the DatasetMetadataRecord's doi field? The prefix is already deleted.

Reply:
Updates made